tests/unit/test_overrides.py

```diff
@@ -56,3 +56,59 @@ def patch_func(*a):
             )
 
         assert stdout_buf.getvalue() != "line 1\nline 2\n"
+
+    def test_types_region_prices_list(
+        self, mock_cli, list_operation_for_overrides_test
+    ):
+        response_json = {
+            "data": [
+                {
+                    "addons": {
+                        "backups": {
+                            "price": {"hourly": 0.008, "monthly": 5},
+                            "region_prices": [
+                                {
+                                    "hourly": 0.0096,
+                                    "id": "us-east",
+                                    "monthly": 6,
+                                }
+                            ],
+                        }
+                    },
+                    "class": "standard",
+                    "disk": 81920,
+                    "gpus": 0,
+                    "id": "g6-standard-2",
+                    "label": "Linode 4GB",
+                    "memory": 4096,
+                    "network_out": 1000,
+                    "price": {"hourly": 0.03, "monthly": 20},
+                    "region_prices": [
+                        {"hourly": 0.036, "id": "us-east", "monthly": 24}
+                    ],
+                    "successor": None,
+                    "transfer": 4000,
+                    "vcpus": 2,
+                }
+            ],
+            "page": 1,
+            "pages": 1,
+            "results": 1,
+        }
+
+        override_signature = ("linodes", "types", OutputMode.table)
+
+        list_operation_for_overrides_test.command = "linodes"
+        list_operation_for_overrides_test.action = "types"
+        mock_cli.output_handler.mode = OutputMode.table
+
+        stdout_buf = io.StringIO()
+
+        with contextlib.redirect_stdout(stdout_buf):
+            list_operation_for_overrides_test.process_response_json(
+                response_json, mock_cli.output_handler
+            )
+
+        rows = stdout_buf.getvalue().split("\n")
+        # assert that the overridden table has the new columns
+        assert len(rows[1].split("┃")) == 15
```

Comment:
Looks like the table characters might be encoded differently on Windows which is causing the GHA tests to fail. 

As a somewhat ugly workaround, you could try resolving the first character in the row and using it to split the table.

e.g.
```python
assert len(rows[1].split(rows[1][0])) == 15
```